### ISSUE TITLE
Bounty: Starjacker stuck at stage 700

### ISSUE THREAD

**Issue by Constellation-VASCO** [Starfield-Community-Patch/Starfield-Community-Patch#1046]
In the starjacker bounty quest, stage 700 has you board the decoy ship. When I board the ship, the stage does not progress and I see an action to sit in the seat. When I sit in the seat, a new activity appears in the upper left side of the screen, as can be seen in this video clip: https://youtu.be/jsCLtpR8pJQ. It reads, "Mission Updated ACTIVITIES Visit the London Landmark On Earth". When I take off and get into orbit of the planet we just left, nothing happens; the game does not progress. I have tried many things but none of them resolve this. I am stuck and cannot play most of this quest.

---

## Details 
In the starjacker bounty quest, stage 700 has you board the decoy ship. When I board the ship, the stage does not progress and I see an action to sit in the seat. When I sit in the seat, a new activity appears in the upper left side of the screen, as can be seen in this video clip: https://youtu.be/jsCLtpR8pJQ. It reads, "Mission Updated ACTIVITIES Visit the London Landmark On Earth". When I take off and get into orbit of the planet we just left, nothing happens; the game does not progress. If you fly to the correct planet, nothing happens. The snitch never speaks or updates the mission. If I load the game onto Windows 10 and look at the stages, it shows all complete except for this stage. If I set stage to 800 in orbit of the planet we left, the snitch then does dialogue and the mission now shows the correct planet to fly to. However, when you fly there, the game still does not progress. When I pull up the console again, it shows that 700 is still incomplete. I can set stage to 810, 850, 900 etc. but none of those make the chop shop visible. I am unable to progress.

I have tried a few things. First, I have disabled all mods and that does not fix the issue. 2nd, I have gone back to my own ship instead of boarding the decoy to try and trigger the London Mission but it does not trigger it. I have gone back several saves and loaded in the frontier (I have had other bugs occur in the game where you had to be in the frontier to progress) and I still experience the same issue. I have gone back weeks in my game saves and still run into the same issue. 

My belief is that the game triggers this new mission instead of triggering stage 800. It then then thinks we are no longer on the other mission even though it shows active. I have never had the london mission come up, ever, in game play and this is the only time I have seen it. I have also tried the london activity but then the snitch is stuck on earth and refuses to leave. I am not aware of a stage completion command other than completing all stages at once.

## Game Version
1.12.32.0

## Language
English (EN)

## New Game
No

## Reproduction Steps
I have not tried it on a new game. I have recreated it multiple times on my saves but haven't tried recreating it other ways.

**Comment by Constellation-VASCO** [comment]
I went through the unity to start NG+ and this resolved the bug. I think overarchingly, my entire game was bugged because I had plenty of other issues including: Mantis ship not going to surface and not being able to board it, UC missions to find and destroy ships and those ships would never spawn, I was never given the star eagle after becoming a freestar marshal, magsniper not working, and plenty of other bugs. I have noticed that in NG+ the magsniper actually works but this play through I didn't try anything other than starjacker.
